ci/run_clang_tidy.sh

```diff
@@ -26,6 +26,13 @@ echo "build ${BAZEL_BUILD_OPTIONS}" >> .bazelrc
 # by clang-tidy
 "${ENVOY_SRCDIR}/tools/gen_compilation_database.py" --run_bazel_build --include_headers
 
+# Do not run clang-tidy against win32 impl
+# TODO(scw00): We should run clang-tidy against win32 impl. But currently we only have 
+# linux ci box.
+function exclude_win32_impl() {
+  grep -v source/common/filesystem/win32/
```

Comment:
exclude other win32 as well? like thread etc.

Reply:
@lizan Thanks and updated.